ui/html/ui-tools.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <title>Lamassu Machine Dev Tools</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <style>
+    body {
+      background-color: #82dda3;
+      font-family: 'MuseoSans', sans-serif;
+    }
+    main {
+      max-width: 42rem;
```

Comment:
Page looks a little bit tight right now.
```suggestion
      max-width: 52rem;
```